cpp/test/neighbors/selection.cu

```diff
@@ -482,6 +482,11 @@ INSTANTIATE_TEST_CASE_P(SelectionTest,
  *  SelectionTest/ReferencedRandomFloatSizeT.LargeK/0
  *  Indicices do not match! ref[91628] = 131.359 != res[36504] = 158.438
  *  Actual: false (actual=36504 != expected=91628 @38999;
+ *
+ *  SelectionTest/ReferencedRandomFloatSizeT.LargeK/1
+ *  ERROR: ref[57977] = 58.9079 != res[21973] = 54.9354
+ *  Actual: false (actual=21973 != expected=57977 @107999;
+ *
```

Comment:
Should we be tracking these in issues?

Reply:
I don't know. @cjnolet do we have an existing issue for the selection test failures?

Reply:
This doesn't have to block this PR. I'll open an issue and we can proceed.

Reply:
https://github.com/rapidsai/raft/issues/1643